XCSummary/Templates/SummaryTemplate.html

```diff
@@ -1,9 +1,9 @@
 <!-- Test Summary -->
 <div class="test-summary" style="margin-left: 10px; background-color: #eeeeee; padding:4px; text-align: right; font-size:20px">
-    <table style="width: 100%">
+    <table style="width: 100%%">
```

Comment:
Percent typo?

Reply:
Surprisingly not a typo! This is the code for percent sign when using `stringWithFormat`: https://stackoverflow.com/a/739707/3023500, so this will ultimately render as a single `%`.